core-java-modules/core-java-collections-list-6/src/test/java/com/baeldung/modifyandprint/ModifyAndPrintListElementsUnitTest.java

```diff
@@ -0,0 +1,56 @@
+package com.baeldung.modifyandprint;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Lists;
+
+public class ModifyAndPrintListElementsUnitTest {
+
+    private final Logger log = LoggerFactory.getLogger(ModifyAndPrintListElementsUnitTest.class);
+
+    @Test
+    void whenPrintingInForEach_thenListIsPrinted() {
+        List<String> theList = Lists.newArrayList("Kai", "Liam", "Eric", "Kevin");
+        theList.forEach(e -> log.info(e));
```

Comment:
In these kinds of cases I usually prefer the log::info method reference syntax

I also prefer words for lambda variable names instead of single letters

Both of these are personal preferences, happy to keep things as-is if you prefer the current approach more

Reply:
In general, both are ok to me. With some FP and Python background, I personally find lambda (just slightly) easier to read. 
But I picked lambda expressions here as we can only use lambda in the map() with print() approach. I thought it might be good not to mix them in the article. Probably, it's an unimportant consideration. 😄 

I renamed the lambda variables from e -> element.